### PR TITLE
FIX + MAINT installation of cask on travis (OSX)

### PR DESCRIPTION
**Pull request by NelleV**
## Description

This is an attempt to fix the OSX travis-CI failures. I'm just trying what travis is suggesting…
Let's see if it works!

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

**Comment by rfezzani** [comment]
In [cask/README.md](https://github.com/Homebrew/homebrew-cask/blob/master/README.md):
> Before reporting a bug, run brew update-reset && brew update and try your command again. This is a fix-all that will reset the state of all your taps, ensuring the problem isn’t an outdated setup on your side.

It's may be something to try...

File: tools/travis/osx_install.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -ex
 
 brew update
 brew install ccache
-brew tap caskroom/cask
+brew tap homebrew/cask-cask
```
Comment (rfezzani) [inline tools/travis/osx_install.sh]:
```suggestion

```
From brew [homepage](https://brew.sh/), it looks like cask is built in... Try to remove this line...

Comment (rfezzani) [reply tools/travis/osx_install.sh]: This fix works, I tested it in #4218 ;-)

**Comment by NelleV** [comment]
Thanks!

**Comment by hmaarrfk** [comment]
@meeseeksdev backport to v0.14.x

**Comment by jni** [comment]
:tada:

**Pull request merged**